integration_tests/requirements.txt

```diff
@@ -17,4 +17,4 @@ pandas
 pyarrow
 pytest-xdist >= 2.0.0
 findspark
-fastparquet >= 2023.8.0
\ No newline at end of file
+fastparquet >= 0.8.3
```

Comment:
`>= 0.8.3` seems inconsistent with below changes in dockerfiles, we want exact 0.8.3 or 0.8.3+ here?

Reply:
Right, gotcha.  I kept it that way, in case someone used `requirements.txt` to install a version that exceeds `0.8.3`. For instance, `2023.8.0` works fine.

That said, I've nailed it down now to `0.8.3`.  I'll see how the tests fare.  I'm also wary of the run on Databricks.